### ISSUE TITLE
Capital-forming benefits

### ISSUE THREAD

**Issue by Mi-si** [portfolio-performance/portfolio#1998]
**Capital-forming benefits**

I receive a monthly allowance of €20 from my employer in addition to my capital-forming benefits.

I would like to be able to book this separately each month, as the view is currently a little distorted. 
Split (if possible) should be best as follows:
Deposit employee
Deposit employer
Deposit state

Is this possible?

-----------------------------------------------

**Vermögenswirksame Leistungen**

Ich erhalte monatlich von meinem Arbeitgeber eine Zulage in Höhe von 20€ zu meinen Vermögenswirksamen Leistungen.

Diese würde ich gerne getrennt monatlich buchen können, da aktuell die Sicht ein wenig verzerrt ist. 
Aufteilung (wenn möglich) sollte am besten wie folgt sein:
Einlage Arbeitnehmer
Einlage Arbeitgeber
Einlage Staat

Ist dies möglich?

**Comment by chirlu** [comment]
I don’t think it makes sense to distinguish between money you earned, money you got from aunt Mary, money you happened to find on the street, etc. It’s your money now either way.

Related (in German): https://forum.portfolio-performance.info/t/vermoegenswirksame-leistungen/2198
